config/config.go

```diff
@@ -263,6 +265,18 @@ type ClientConfig struct {
 	// <HOST>:<PORT> of the user's server (e.g. Rails server)
 	LocalEndpoint string
 
+	// LocalUseTLS allows us to specify whether to connect to local
+	// Note: this is for wh-client <-> local-endpoint only
+	LocalEndpointUseTLS bool
+
+	// LocalInsecureSkipVerify disables SSL cert verification for local endpoints
```

Comment:
super minor, but probably the var names in comments should match the actual var names (for the lint's sake at least)?

Reply:
Ah whoops. Yeah, typo